tests/test_efadv.py

```diff
@@ -5,9 +5,12 @@
 """
 
 import errno
+from pyverbs.addr import AHAttr
 from pyverbs.base import PyverbsRDMAError
+from pyverbs.pd import PD
 import pyverbs.providers.efa.efadv as efa
 from tests.base import PyverbsAPITestCase
+from tests.test_addr import get_global_route
```

Comment:
I think get_global_route() can be moved to "tests.utils".

Reply:
Ack